src/App.js

```diff
@@ -1,20 +1,37 @@
-import { useState } from 'react'
+import { useEffect, useState } from 'react'
 import { Route, Switch } from 'react-router-dom'
-import movies from './movieList.json'
 
 import SearchPage from './components/SearchPage'
 import FilteredMoviesPage from './components/FilteredMoviesPage'
 import HomePage from './components/HomePage'
+require('dotenv').config()
 
-function App() {
-  const [genres, setGenres] = useState([])
+export default function App() {
   const [players, setPlayers] = useState([])
+  const [fetchMovies, setFetchMovies] = useState([])
+  const [genres, setGenres] = useState([])
+  const [filterByGenres, setFilterByGenres] = useState([])
 
-  const filteredMovies = movies.filter(
-    movie => genres.length === 0 || movie.genre.some(g => genres.includes(g))
-  )
+  const MOVIE_API = `https://api.themoviedb.org/3/discover/movie?sort_by=popularity.desc&api_key=${process.env.REACT_APP_TMDB_API_KEY}`
+  const GENRE_API = `https://api.themoviedb.org/3/genre/movie/list?api_key=${process.env.REACT_APP_TMDB_API_KEY}&language=en-US`
+
+  useEffect(() => {
+    fetch(MOVIE_API)
+      .then(res => res.json())
+      .then(data => {
+        console.log(data)
+        setFetchMovies(data.results)
+      })
```

Comment:
You should consider a `.catch` in case there is an error. And build in error handling.
```suggestion
      }).catch(setError)
```